src/node_buffer.cc

```diff
@@ -209,6 +209,8 @@ inline MUST_USE_RESULT bool ParseArrayIndex(Local<Value> arg,
 
   // Check that the result fits in a size_t.
   const uint64_t kSizeMax = static_cast<uint64_t>(static_cast<size_t>(-1));
+  // Tell coverity that this is not a constant expression on all platforms.
```

Comment:
It _is_ a constant expression, however.  I'd just leave out the comment, it's clear enough from context that we're trying to shut up coverity.